src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -79,12 +79,13 @@
 (defn content-title
   ([title] (content-title nil title))
   ([title-style title]
-   (content-title title-style title nil nil))
-  ([title-style title subtitle-style subtitle]
+   (content-title title-style title nil nil nil))
+  ([title-style title subtitle-style subtitle additional-text-props]
```

Comment:
what if i call `(content-title title-style title subtitle-style subtitle)` ?

Reply:
it should then fallback to `(content-title title-style title subtitle-style subtitle nil)` which then execute the function without altering the prop hashmap, right?

Reply:
yes we need to add `[title-style title subtitle-style subtitle]` here

Reply:
ahh yes, I forgot to implement the 4 param version to be retrocompatible... i'll updating asap...